src/coreclr/pal/tests/palsuite/compilableTests.txt

```diff
@@ -1,18 +1,3 @@
-c_runtime/abs/test1/paltest_abs_test1
```

Comment:
Just noting that I believe every single one of these tests is covered in managed for the exposed Math/MathF APIs, so we shouldn't be losing any coverage

Reply:
Yup, but the primary purpose of PAL tests is to help aid new platform ports, we start with PAL and work our way up to VM, JIT and native libs to run managed lib. These test ensure that the 'OS API is the shape which .NET Runtime expects'. By the time we are able to execute managed code, most of these tests are pretty much redundant from 'coverage' perspective.

Reply:
Yup, but the primary purpose of PAL tests is to help aid new platform ports, we start with PAL and work our way up to VM, JIT and native libs to run managed code. These tests ensure that the 'OS API is the shape which .NET Runtime expects'. By the time we are able to execute managed code, most of these tests are pretty much redundant from 'coverage' perspective.